inc/class-provider.php

```diff
@@ -35,6 +35,26 @@ public function __construct( Factory $factory ) {
 		$this->factory = $factory;
 	}
 
+	/**
+	 * Return the provider ID.
+	 *
+	 * @return string
+	 */
+	public function get_id(): string {
+
+		return 'wp';
```

Comment:
I honestly would like to see `'wordpress'` here, given that all references to WordPress are using the full name. AMF WordPress. I think I'm going to exclude the sniff from this repo's PHPCS config.